src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -412,6 +412,20 @@ protected void writeExtractCheckFile(PaymentStatus extractedStatus, PaymentProce
         }
     }
 
+    private String createFormattedRemittanceIdTextBasedOnPresenceOfInvoiceDataForDv(String invoiceNumber, Date invoiceDate, String customerPaymentDocumentNumber) {
+        StringBuffer formattedRemittanceIdText = new StringBuffer("");
+
+        if (StringUtils.isNotBlank(invoiceNumber) && ObjectUtils.isNotNull(invoiceDate)) {
+            formattedRemittanceIdText.append(invoiceNumber);
+            formattedRemittanceIdText.append(" Doc No:");
```

Comment:
For the combined string that has both invoice number and document number, I believe Marcia wanted the doc number label to be changed to "Doc:" instead. If needed, you can confirm with Marcia whether this label should be "Doc No:" or just "Doc:". (I think the remittance ID field has a large enough max length to accommodate either option.)

Reply:
Marcia has confirmed we can use "Doc:" as the label for both this change and the existing code paths when there is no invoice number or date. Fixing.